comid/comid.go

```diff
@@ -214,10 +214,10 @@ func (o *Comid) AddEndorsedValue(val ValueTriple) *Comid {
 
 // AddAttestVerifKey adds the supplied endorsed value to the
 // attest-key-triples list of the target Comid.
-func (o *Comid) AddAttestVerifKey(val AttestVerifKey) *Comid {
+func (o *Comid) AddAttestVerifKey(val KeyTriple) *Comid {
```

Comment:
While you are at 217 may be please correct the comment on Line 215, it is perhaps a `Verification key` rather than an Incorrect `endorsed value` due to a copy paste error